packages/modules/viewer-react/package.json

```diff
@@ -33,31 +33,34 @@
     "@itwin/itwinui-illustrations-react": "^2.0.1",
     "@itwin/itwinui-react": "^2.6.0",
     "@itwin/reality-data-client": "^1.0.0",
+    "@itwin/unified-selection": "^0.4.4",
     "lodash.isequal": "^4.5.0"
   },
   "devDependencies": {
-    "@itwin/appui-abstract": "^4.0.0",
-    "@itwin/appui-layout-react": "^4.0.0",
-    "@itwin/appui-react": "^4.0.0",
```

Comment:
I know appui has killed off appui-layout-react, but since we still techniclaly list it as a peer dep, should we be removing the dev dep on it?

Reply:
And this propagates through the other viewer pkgs

Reply:
Right, reverted the changes to appui dependencies.